src/utils/parse-arguments-into-options.ts

```diff
@@ -1,11 +1,30 @@
-import type { Args, RawOptions } from "../types";
+import type { Args, ExternalExtension, RawOptions } from "../types";
 import arg from "arg";
 import * as https from "https";
 import { getDataFromExternalExtensionArgument } from "./external-extensions";
 import chalk from "chalk";
 import { CURATED_EXTENSIONS } from "../config";
+import fs from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
+
+const validateTemplate = async (
+  template: string,
+  dev: boolean,
+): Promise<{ repository: string; branch?: string } | string> => {
+  if (dev) {
+    // Check that the template folders exists in extensions/${template}
+    try {
+      const currentFileUrl = import.meta.url;
+      const extensionsDirectory = path.resolve(decodeURI(fileURLToPath(currentFileUrl)), "../../externalExtensions");
```

Comment:
I think this should be called `externalExtensionDirectory` ? After we planned to change the naming

Reply:
Yep!!  https://github.com/scaffold-eth/create-eth/pull/57/commits/d98e627c0fd382bc1b71f9896d65c91f72e54e46